buster/utils.py

```diff
@@ -34,3 +33,14 @@ def get_documents_manager_from_extension(filepath: str) -> Type[DocumentsManager
         return DocumentsDB
     else:
         raise ValueError(f"Unsupported format: {ext}.")
+
+
+def get_retriever_from_extension(filepath: str) -> Type[Retriever]:
```

Comment:
i dont think you need to explicitly mention `Type` here in the typehints, maybe just `-> Retriever` should be enough?